fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -3043,7 +3048,13 @@ partition_key_desc ::=
 
 /* list partition PartitionKeyDesc */
 list_partition_key_desc ::=
-    LPAREN list_partition_values_list:keys RPAREN
+    /* empty */
```

Comment:
How about adding `DEFAULT`?

Reply:
> How about adding `DEFAULT`?

Sounds great. It'll be more readable for user. I'll change it soon.

Reply:
Not changed?